src/main/java/com/wanderersoftherift/wotr/init/ModItems.java

```diff
@@ -92,4 +89,12 @@ public static <T extends Block> DeferredItem<BlockItem> registerSimpleBlockItem(
         return simpleBlockItem;
     }
 
+    public static <T extends Block> DeferredItem<BlockItem> registerSimpleDevBlockItem(
```

Comment:
Not entirely satisfied with this change, as this piece of code can be used for all blocks.

Reply:
Nvm, did not see the original method still exists.